src/NuGet.Core/NuGet.Build.Tasks/Common/MSBuildLogger.cs

```diff
@@ -1,4 +1,7 @@
-﻿using Microsoft.Build.Framework;
+﻿// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
nit:
Since these files have nothing to do with your change, but you've already done the 'tedious' work, I'd cherry pick the respective commits into a different PR and keep this as minimal as possible :)